dist/seamly2d.dsc

```diff
@@ -4,15 +4,15 @@ Binary: seamly2d
 Architecture: i386 amd64
 Version: 0.3.3
 Maintainer: Susan Spencer <susan.spencer@gmail.com>
-Homepage: https://fashionfreedom.eu/
+Homepage: https://seamly.net
 Standards-Version: 3.9.5
 Vcs-Browser: https://github.com/fashionfreedom/seamly2d
 Build-Depends: debhelper (>= 8.0.0), qtbase5-dev (>= 5.0.0), libqt5svg5-dev (>= 5.0.0), g++ (>= 4.6.0), qt5-default (>= 5.0.0), qttools5-dev-tools (>= 5.0.0), libqt5xmlpatterns5-dev (>= 5.0.0)
-Package-List: 
+Package-List:
  seamly2d deb graphics optional
-Checksums-Sha1: 
+Checksums-Sha1:
  581eb1bf36b4ab7126b5983d809130f15396859e 24838101 seamly2d_0.3.3.tar
-Checksums-Sha256: 
+Checksums-Sha256:
  9b156c7120a69b90373efb8ca9998c3e0563a60ad337210166cfd41b00b0f13c 24838101 seamly2d_0.3.3.tar
-Files: 
+Files:
```

Comment:
Even the version tag is very out-of-date here. We don't rely on these files anymore to produce our product.